execute/plugin_test.go

```diff
@@ -1380,3 +1383,105 @@ func TestPlugin_ShouldTransmitAcceptReport_Failure_AlreadyExecuted(t *testing.T)
+			Observer:    0,
+		},
+	}
+	ctx := tests.Context(t)
+	p := &Plugin{
+		lggr:         logger.Test(t),
+		ocrTypeCodec: jsonOcrTypeCodec,
+		destChain:    3478487238524512106,
+		observer:     &metrics.Noop{},
+	}
+
+	prevOutcomeBytes := []byte(`{"State":"Initialized","commitReports":[],"report":{"chainReports":[]}}`)
+
+	outcomeBytes, err := p.Outcome(ctx, ocr3types.OutcomeContext{
+		SeqNr:           190942,
+		PreviousOutcome: prevOutcomeBytes,
+	}, nil, attObs)
+	require.NoError(t, err)
+
+	//ocrTypeCodec := ocrtypecodec.NewExecCodecJSON()
+	decodedOutcome, err := ocrTypeCodec.DecodeOutcome(outcomeBytes)
+	require.NoError(t, err)
+	// assert uniqueness of merkle roots in commitreports
+	merkleRoots := make(map[string]struct{})
+	for _, commitReport := range decodedOutcome.CommitReports {
+		merkleRoots[commitReport.MerkleRoot.String()] = struct{}{}
+	}
+	require.Equal(t, len(decodedOutcome.CommitReports), len(merkleRoots))
```

Comment:
```suggestion
	require.Equal(t, len(merkleRoots), len(decodedOutcome.CommitReports))
```

Expected comes first, then actual